pkgs/applications/blockchains/nearcore/default.nix

```diff
@@ -1,32 +1,46 @@
 { rustPlatform, lib, fetchFromGitHub
-, zlib, elfutils, openssl
-, cmake, python3, pkg-config, protobuf, perl, llvmPackages
+, zlib, openssl
+, pkg-config, protobuf, llvmPackages
 }:
 rustPlatform.buildRustPackage rec {
-  #https://github.com/near/nearcore
   pname = "nearcore";
-  version = "1.25.0";
+  version = "1.26.0-rc.3";
+
+  # https://github.com/near/nearcore/tags
   src = fetchFromGitHub {
     owner = "near";
     repo = "nearcore";
     # there is also a branch for this version number, so we need to be explicit
     rev = "refs/tags/${version}";
-    sha256 = "sha256-7hiBqJLGIf+kNKJvMQ7KtGZm/SWLY3pT7YDlwbm3HDM=";
+    sha256 = "sha256-MjeCGTS9UPCfk4Wyj1yx0FnsVk7o8b/Ahf9Ex+7MUgw=";
   };
 
-  cargoSha256 = "sha256-EGv4CibSHL9oTAdWK7d/SOzZWPcEB16hTWlWHjKU4wc=";
+  cargoSha256 = "sha256-nadJn8irq7ZsXFXaYIy2ub0ZdDKhfFBaOaH/DOZTUVk=";
+
+  postPatch = ''
+    substituteInPlace neard/src/main.rs \
+      --replace 'git_version!(fallback = "unknown")' '"nix:${version}"' \
+      --replace 'use git_version' '//use git_version'
+  '';
+
+  CARGO_PROFILE_RELEASE_CODEGEN_UNITS = "1";
+  CARGO_PROFILE_RELEASE_LTO = "thin";
```

Comment:
One more thing - nearcore Makefile is using [fat LTO](https://github.com/near/nearcore/blob/c164b7df5f94f54c8f36305348dcd9f9b9994ddc/Makefile#L2=) officially. Might want to match that as well?

Reply:
How much memory does require? I don't want reviewers machine / builds to go out of memory.

Reply:
~3644M on my machine.

Reply:
I enabled it. Does not seem to heavy.